CMakeLists.txt

```diff
@@ -37,6 +37,7 @@ option(NXT_USE_WERROR "Treat warnings as error (useful for CI)" 0)
 # Default values for the backend-enabling options
 set(ENABLE_D3D12 OFF)
 set(ENABLE_METAL OFF)
+set(ENABLE_METAL OFF)
```

Comment:
`ENABLE_VULKAN`?

Reply:
Whoops, forgot to remove it, thanks for the catch!